tests/rewards/test_reward_nets.py

```diff
@@ -531,25 +535,50 @@ def test_add_std_reward_wrapper(
     assert np.allclose(rewards, 1 - 0.5 * np.sqrt(8))
 
 
-@pytest.mark.parametrize("make_wrapper", MAKE_BASIC_REWARD_NET_WRAPPERS)
+def test_shaped_reward_net(
+    zero_reward_net: MockRewardNet,
+    numpy_transitions: NumpyTransitions,
+):
+    def potential(x: th.Tensor):
+        return th.full((x.shape[0],), 10, device=x.device)
+
+    shaped = reward_nets.ShapedRewardNet(zero_reward_net, potential, 0.9)
+    shaped_rew = th.full((10,), -1, dtype=th.float32)
```

Comment:
add comment explaining why we should expect reward to be `-1`? (I had to think about it for a minute, and many people are less familiar with potential shaping than me.)